gradle.properties

```diff
@@ -1,9 +1,8 @@
 groupId=org.eclipse.edc
 edcVersion=0.0.1-milestone-8
 defaultVersion=0.0.1-SNAPSHOT
-javaVersion=11
+javaVersion=17
 
 # configure the build:
 annotationProcessorVersion=0.0.1-SNAPSHOT
-edcGradlePluginsVersion=0.0.1-SNAPSHOT
-metaModelVersion=0.0.1-SNAPSHOT
\ No newline at end of file
+edcGradlePluginsVersion=0.0.1-SNAPSHOT
```

Comment:
please switch this and `annotationProcessorVersion` usages to `edcVersion`, this should fix all the issues